theano/tensor/extra_ops.py

```diff
@@ -1116,34 +1116,64 @@ class Unique(theano.Op):
     >>> import theano
 
     >>> x = theano.tensor.vector()
-    >>> f = theano.function([x], Unique(True, True, False)(x))
+    >>> f = theano.function([x], Unique(True, True, False, None)(x))
```

Comment:
The last `None` is not a mandatory argument, so you don't need to include it in old examples.

Reply:
I'll remove it